site/content/how-to/monitoring/tracing.md

```diff
@@ -13,6 +13,10 @@ NGINX Gateway Fabric supports tracing using [OpenTelemetry](https://opentelemetr
 
 This guide explains how to enable tracing on HTTPRoutes using NGINX Gateway Fabric. It uses the OpenTelemetry Collector and Jaeger to process and collect the traces.
 
+{{< important >}}
+Tracing cannot be enabled for HTTPRoutes that have [HTTPRouteMatches](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.HTTPRouteMatch) with `headers`, `params`, or `method` matchers defined.
```

Comment:
Should we also mention that this will be fixed in a future release? (for both docs)

Just want the language to be clear that this is a bug, not by design.

Reply:
Sure. Something like: "This will be addressed in a future release"?

Reply:
Or "This is a known issue that will be fixed in a future release".

Reply:
The first one sounds good.

Reply:
Should we also say `matches` with those conditions? Because it should work for any paths in the HTTPRoute otherwise that don't have the matches.

Reply:
> Tracing cannot be enabled for [HTTPRoute matches](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.HTTPRouteMatch) with `headers`, `params`, or `method` matchers defined. This will be addressed in a future release.

?

Reply:
Does that make sense?

Reply:
Sounds good.